src/platforms/native/configuration/backends/crashpad.mdx

```diff
@@ -37,6 +37,11 @@ according to shell lookup rules at runtime.
 It is advised to specify both the path to the `crashpad_handler` executable
 as well as the path to the database directory manually.
 
+Multi-process use cases are also supported. Initialize Sentry individually
+in each process which spawns its own Crashpad handler either in a separate
+database path, or using a shared database path. Each run uses its individual
+run directory, and there is filesystem-level locking in place.
```

Comment:
```suggestion
You can also use Crashpad for multi-process use cases. To do this, initialize Sentry individually
in each process that spawns its own Crashpad handler, either in a separate
database path or using a shared database path. Each run will use its individual
run directory, and there is filesystem-level locking in place.
```

Reply:
I found this original sentence to be a little confusing, so hopefully this reword makes it more clear and accurately reflects what you meant!

Reply:
Thanks for the suggestion! I adapted it a tiny bit, wdyt?

Reply:
This looks awesome, thanks for taking the time to make sure this is really clear!